src/openrct2/scripting/bindings/entity/ScGuest.cpp

```diff
@@ -12,9 +12,138 @@
 #    include "ScGuest.hpp"
 
 #    include "../../../entity/Guest.h"
+#    include "../../../localisation/Localisation.h"
 
 namespace OpenRCT2::Scripting
 {
+    static const DukEnumMap<PeepThoughtType> ThoughtTypeMap({
```

Comment:
Can this be `constexpr`ed?

Reply:
EnumMap in its current state can't be `constexpr`ed from what I can tell, and all other instances of it are not marked `constexpr`

Reply:
:( That's unfortunate